packages/admin/cms-admin/src/dam/DataGrid/duplicatedFilenames/ManualDuplicatedFilenamesHandler.tsx

```diff
@@ -23,13 +23,16 @@ export const useManualDuplicatedFilenamesHandler = (): ManualDuplicatedFilenames
     return useContext(ManualDuplicatedFilenamesHandlerContext);
 };
 
-export const ManualDuplicatedFilenamesHandlerContextProvider = ({ children }: { children?: ReactNode }) => {
+export type DuplicateAction = "replace" | "copy" | null;
```

Comment:
Maybe use "skip" instead of null?

Reply:
bb58c9567b04386c1633f1ece4e90789292b9dbb